apps/src/dropletUtils.js

```diff
@@ -712,6 +721,37 @@ function getParamFromCodeAtIndex(index, methodName, code) {
   return null;
 }
 
+/**
+ * Given a stating config, generates a "disabled" droplet config that has the
+ * following properties:
+ *  - Removes all categories
+ *  - Removes all blocks from categories
+ *  - Turns all blocks gray
+ *  - Removes all blocks from autocomplete
+ *  - Removes all additional defines
+ * @param {DropletConfig} originalConfig
+ */
+module.exports.makeDisabledConfig = function makeDisabledConfig(originalConfig) {
```

Comment:
Note: I'd like to do a follow-up PR that updates dropletUtils to ES6-style exports; I tried doing that as part of this PR but I think it's best kept separate.